aip/parser.js

```diff
@@ -68,7 +70,45 @@ module.exports = new Parser(function analyseEC(parsedUrl) {
       break;
     }
 
+  } else if ((match = /^\/aip\/([a-z]+)\/article-pdf\/doi\/((10\.[0-9]+\/([a-z0-9.]+))\/[0-9]+\/[0-9._]+\.pdf)$/i.exec(path)) !== null) {
+    // https://pubs.aip.org/aip/acp/article-pdf/doi/10.1063/5.0164351/18116273/020004_1_5.0164351.pdf
+    result.rtype = 'ARTICLE';
+    result.mime  = 'PDF';
+    result.title_id = match[1];
+    result.doi = match[3];
+    result.unitid = match[4];
+  } else if ((match = /^\/aip\/([a-z]+)\/article-pdf\/[0-9]+\/([0-9._]+)\.pdf$/i.exec(path)) !== null) {
+    // https://pubs.aip.org/aip/acp/article-pdf/18146317/020023_1_5.0167690.pdf
+    result.rtype = 'ARTICLE';
+    result.mime  = 'PDF';
+    result.title_id = match[1];
+    result.unitid = match[2];
+  } else if ((match = /^\/aip\/([a-z]+)\/article\/([0-9]+)\/([0-9]+)\/([0-9]+)\/([0-9]+)\/([a-z-]+)$/i.exec(path)) !== null) {
+    //https://pubs.aip.org/aip/acp/article/2872/1/120047/2913626/Brain-activity-vs-seismicity-Scaling-and-memory?searchresult=1
+    result.rtype = 'ARTICLE';
+    result.mime  = 'HTML';
+    result.title_id = match[1];
+    result.vol  = match[2];
+    result.issue = match[3];
+    result.unitid = match[5];
```

Comment:
```suggestion
  } else if ((match = /^\/aip\/([a-z]+)\/article\/([0-9]+)\/([0-9]+)\/[0-9]+\/([0-9]+)\/[a-z-]+$/i.exec(path)) !== null) {
    //https://pubs.aip.org/aip/acp/article/2872/1/120047/2913626/Brain-activity-vs-seismicity-Scaling-and-memory?searchresult=1
    result.rtype = 'ARTICLE';
    result.mime  = 'HTML';
    result.title_id = match[1];
    result.vol  = match[2];
    result.issue = match[3];
    result.unitid = match[4];
```

Isn't there too much matching groups here ?